torch/utils/model_zoo.py

```diff
@@ -68,9 +68,11 @@ def load_url(url, model_dir=None, map_location=None, progress=True):
 
 
 def _download_url_to_file(url, dst, hash_prefix, progress):
+    file_size = None
     if requests_available:
         u = urlopen(url, stream=True)
-        file_size = int(u.headers["Content-Length"])
+        if hasattr(u.headers, "Content-Length"):
+            file_size = int(u.headers["Content-Length"])
```

Comment:
You should also fix the case below (no `requests` package)

Reply:
those dont need to be fixed because they already return `None` on a missing field. i.e. get_all("Content-Length") or getheaders("Content-Length")

Reply:
We call `meta.get_all("Content-Length")[0]` -- won't the `[0]` fail?